src/mode/modeNormal.ts

```diff
@@ -64,7 +64,7 @@ export default class NormalMode extends Mode {
 
             if (keyHandled) {
                 this.keyHistory = [];
-                return this.keyHandler[keysPressed](this.Motion);

```

Comment:
I think this goes away with the async/await stuff that was just merged a couple of days ago.